quickannotator/populate_db.ipynb

```diff
@@ -214,16 +213,24 @@
     "        c = Point(np.random.randint(image_width), np.random.randint(image_height))\n",
     "        \n",
     "        poly: Polygon = generate_random_polygon(max_area=10000, centroid=(c.x, c.y))\n",
-    "        d = {\n",
-    "            \"image_id\": image_id,\n",
-    "            \"annotation_class_id\": annotation_class_id,\n",
-    "            \"isgt\": gtpred == \"gt\",\n",
-    "            \"centroid\": poly.centroid.wkt,  # Adding SRID=0 for pixel-based coordinates\n",
-    "            \"area\": poly.area,  # The area of the polygon\n",
-    "            \"polygon\": poly.wkt,  # Adding SRID=0 for the polygon\n",
-    "            \"custom_metrics\": json.dumps({\"iou\": 0.5})  # Convert custom_metrics to a JSON string\n",
-    "        }\n",
-    "        annotations.append(d)\n",
+    "        x = poly.centroid.x\n",
+    "        y = poly.centroid.y\n",
+    "        tiles = tiles_within_bbox(db, image_id, annotation_class_id, x, y, x, y)\n",
+    "        if len(tiles) == 0:\n",
+    "            annotations.append(None)\n",
+    "        else:\n",
+    "            tile_id = tiles[0].id\n",
+    "            d = {\n",
+    "                \"image_id\": image_id,\n",
+    "                \"annotation_class_id\": annotation_class_id,\n",
+    "                \"isgt\": gtpred == \"gt\",\n",
+    "                \"centroid\": poly.centroid.wkt,  # Adding SRID=0 for pixel-based coordinates\n",
```

Comment:
do we need to store the centroid or should it be computed dynamically by the backend?